CODEOWNERS

```diff
@@ -539,6 +540,7 @@
 /share/zephyr-package/                    @tejlmand
 /share/zephyrunittest-package/            @tejlmand
 /subsys/bluetooth/                        @joerchan @jhedberg @Vudentz
+/subsys/bluetooth/audio/                  @Vudentz @Thalley @asbjornsabo
```

Comment:
Is this right? jhedberg isn't added here, but the files is put under the generic bluetooth which is maintained by jhedberg.
Should there be an additional maintainers section for bluetooth/audio?

Commit message also says additional, which might suggest that you wanted to just do addition?

Reply:
This may be me not knowing the interpretation of this syntax.

I added the bluetooth/audio/, to limit the scope for me and Emil as maintainers.
I added to those directories those that I know work with le-audio.  I also assumed that those listed for "higher" levels automatically would have rights for levels below, so that jhedberg and you would have rights anyway.
(I have not been consistent, though, because than it would not be necessary to add Vudentz to the audio directory, since he also is on the level above.)

The intention is basically
- Those of you who are bluetooth code owners continue to be so (over all of bluetooth)
- Emil and I get codeowner status in the audio subset of bluetooth.

Reply:
In that case you need to add me and jhedberg.
I'm not being pulled in to review /subsys/bluetooth/controller/ .

Reply:
Updated to include all previous code owners also for the audio subdirectories.